eng/pipelines/libraries/enterprise/linux.yml

```diff
@@ -26,7 +26,7 @@ variables:
   - name: enterpriseTestsSetup
     value: $(sourcesRoot)/Common/tests/System/Net/EnterpriseTests/setup
   - name: containerRunTestsCommand
-    value: /repo/.dotnet/dotnet build /t:test --no-restore
+    value: /repo/.dotnet/dotnet build /t:test
```

Comment:
Why? Tests will already be restored at the build phase, no?

Reply:
No, only if `/p:IncludeLibrariesTestSubset=true` is specified, which we don't want here as we only care about two test projects.

Reply:
Ok, that's great! I hated the fact that the tests were restored if I wanted to just build the product.